src/System.Net.HttpListener/src/System/Net/CaseInsensitiveAsciiComparer.cs

```diff
@@ -0,0 +1,131 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+
+namespace System.Net
+{
+    internal class CaseInsensitiveAsciiComparer : IEqualityComparer, IComparer
```

Comment:
How does this differ from the CaseInsensitiveAscii comparer already in the repo?
[https://github.com/dotnet/corefx/blob/master/src/System.Net.WebHeaderCollection/src/System/Net/WebHeaderCollection.cs#L496](https://github.com/dotnet/corefx/blob/master/src/System.Net.WebHeaderCollection/src/System/Net/WebHeaderCollection.cs#L496)
If possible, we should combine them to avoid duplication.